core/src/main/java/org/elasticsearch/action/admin/indices/analyze/AnalyzeRequest.java

```diff
@@ -124,6 +147,9 @@ public ActionRequestValidationException validate() {
         if (charFilters == null) {
             validationException = addValidationError("char filters must not be null", validationException);
         }
+        if (attributes == null) {
+            validationException = addValidationError("attributes must not be null", validationException);
+        }
```

Comment:
Given that attributes has a sensible default, can we not check on the setter method that the parameter is non null rather than checking here, so that way we can fail faster?